AppBundles/ExportDrawingAsPdfPlugin/Automation.cs

```diff
@@ -37,64 +37,27 @@ public override void ExecWithArguments(Document doc, NameValueMap map)
             using (new HeartBeat())
             {
                 var dir = System.IO.Directory.GetCurrentDirectory();
-                var drawingToGenerate = map.Count>1 ? map.Item["_2"] : null;
+                var drawingToGenerate = map.Item["_2"];
 
                 if (drawingToGenerate == null)
                 {
                     LogTrace("Drawing not specified !");
```

Comment:
@vasicem prefers to fail right away, right?

Reply:
I mean - before we wrote a log trace "Drawing is not specified" and quit.  Now (I suppose) the code will fail on line 49:
https://github.com/Autodesk-Forge/forge-configurator-inventor/blob/da0b54256c2566ad206a7f600861e0e07350759b/AppBundles/ExportDrawingAsPdfPlugin/Automation.cs#L49
Or maybe it will fail later.

IMHO - it's better to quit without failing with explanation in report.

Reply:
> IMHO - it's better to quit without failing (with some explanation in report).
I think it is not better. When you use activity bad, you want to know it ASAP. It is programmatic issue, not runtime issue.

Reply:
So instead of "Drawing is not specified" you'll have something like:
```
Run-time exception (line 8): Value cannot be null.
Parameter name: path2

Stack Trace:

[System.ArgumentNullException: Value cannot be null.
Parameter name: path2]
   at System.IO.Path.Combine(String path1, String path2)
   at Program.Main() :line 8
```

why not to throw something meaningful here? (even if it's programmatic issue)

Reply:
Yes, definitely lets throw something meaningful or write something meaningful before we throw ... but lets throw. I mean we need to make sure that such workitem is marked as failed. That was my point.